www/admin/inner.js

```diff
@@ -1476,10 +1481,18 @@ Example
 
         var input = h('input', {
             type: 'email',
-            value: ApiConfig.adminEmail || ''
+            value: ApiConfig.adminEmail || '',
+            id: 'emailInput'
         });
         var $input = $(input);
-        var innerDiv = h('div.cp-admin-setter.cp-admin-setlimit-form', input);
+
+        Messages.email = "Email:"; // XXX temporary solution
+        var label = h('label', {
+            for: 'emailInput'
+        },Messages.email);
+
+        var innerDiv = h('div.cp-admin-setter.cp-admin-setlimit-form', [label, input]);
```

Comment:
The label and input could be combined in a more legible way 

![image](https://github.com/cryptpad/cryptpad/assets/11028490/5d80359b-b406-4cfe-95a9-266c169b60bf)